python-crud-google-spreadsheet/src/create.py

```diff
@@ -0,0 +1,18 @@
+import gspread
+from oauth2client.service_account import ServiceAccountCredentials
+import os
+
+
+def handler(body, req):
```

Comment:
A docstring would be very useful for defining the expected payload, the prerequisites (e.g. an existing sheet with a worksheet called `Sheet`), and the outcome (what is created? A worksheet? A new row?)